packages/client/src/payments/types/paymentMethods.types.ts

```diff
@@ -23,6 +23,8 @@ export type CreditCard = {
 export type CreditCardPaymentMethod = {
   type: string;
   creditCards: CreditCard[];
+  supportsInstallments: boolean;
```

Comment:
Please add `?` to `creditCards`, `supportsInstallments` and `installments` since they are optional

Reply:
updated